sentry-core/src/main/java/io/sentry/core/protocol/SdkVersion.java

```diff
@@ -53,4 +54,12 @@ public void addIntegration(String integration) {
   public void acceptUnknownProperties(Map<String, Object> unknown) {
     this.unknown = unknown;
   }
+
+  public @Nullable List<SentryPackage> getPackages() {
+    return packages;
+  }
+
+  public @Nullable List<String> getIntegrations() {
+    return integrations;
+  }
```

Comment:
I agree that SdkVersion is not the best name, but SdkInfo is a different object nominated by the unified API, we should not mix them.

I'd not rename SdkVersion now because it'll be a breaking change, we can do this on 3.0 if it's the case.
But totally against of adding those things to SdkInfo as this is a totally different object, we discussed about this on TSC

Reply:
Ah okay, I get it because of the breaking change.